docs/user/technical-reference/README.md

```diff
@@ -0,0 +1,3 @@
+# Technical reference
```

Comment:
I think that under technical-reference directory we are missing the dedicated page describing Application-Connectivity-Validator service as one of the elements of Application Connector Module.

Reply:
I just think it deserves it own Technical Reference Page. On the main page it is has only one description.